release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1194,6 +1198,16 @@ With this update, the `tunbr` interface is now included in the list of interface
 [id="ocp-4-11-node-bug-fixes"]
 ==== Node
 
+* Previously, the pod manager handled the registration and de-registration  of pod secrets and config maps. Because of this, pod secrets would sometimes fail to be mounted within a pod. With this fix, the pod id is included in the key that kublet uses to manage registered pods. As a result, secrets are proper mounted as expected. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1999325[*BZ#1999325*])
+
+* Because of a memory leak in the garbage collection process, pods might not be able to start on a node due to lack of memory. With this fix  memory is no longer leaking in the garbage collection prodess and nodes should start as expected. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2065749[*BZ#2065749*])
+
+* Because of a change in the upstream Kubernetes, the kubelet was not running readiness probes on terminated pods. As a result, a load balancer or controller could react more slowly to a terminating pod, which might have resulted in errors. With this fix, readiness probes are again being performed on pod termination. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2089933[*BZ#2089933*])
+
+* Because of a bug, kubelet could incorrectly reject pods that have `OutOfCpu` errors, if the pods were rapidly scheduled after other pods were reported as complete in the API. With this fix, the Kubelet now waits to report the phase of a pod as terminal in the API until all running containers have stopped and no new containers have been started. Short-lived pods may take slightly longer (~1s) to report success or failure after this change. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2022507[*BZ#2022507*])
+
+* Because recent versions of the `prometheus-adapter` are sending additional pod metrics, the Vertical Pod Autoscaler (VPA) recommender is producing a large number of unnecessary and repetitive messages. With this fix,  the VPA recognizes and ignores the extra metrics. As a result, these messages are no longer being produced. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2102947[*BZ#2102947*])
```

Comment:
```suggestion
* Because recent versions of the `prometheus-adapter` are sending additional pod metrics, the Vertical Pod Autoscaler (VPA) recommender is producing a large number of unnecessary and repetitive messages. With this fix, the VPA recognizes and ignores the extra metrics. As a result, these messages are no longer being produced. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2102947[*BZ#2102947*])
```